### ISSUE TITLE
Placing IR parts with symmentry causes them to not function

### ISSUE THREAD

**Issue by Vilkku** [MagicSmokeIndustries/InfernalRobotics#98]
Apologies if this is a known issue, I could not find references to this. Using the latest release (2.0.0), did not have a MagicSmokeIndustries directory before installing. Installed both legacy parts and core, tried installing core first and then legacy parts after that, later on tried to reverse the order as there were some files that were present in both packages.
### Versions
- KSP 1.1.2.1260 (x64) (Steam)
- [Infernal Robotics 2.0 Final](https://github.com/MagicSmokeIndustries/InfernalRobotics/releases/tag/2.0.0)
- [IR-LegacyParts.zip](https://github.com/MagicSmokeIndustries/InfernalRobotics/releases/tag/2.0.0)
#### Other mods
- MechJeb2
- Kerbal Engineer Redux
### Steps to reproduce
1. Create vessel in SPH
2. Activate symmetry (defaulting to mirror)
3. Place IR Rotatron somewhere on the vessel (since symmetry is activated, two of them will be placed, as expected)
4. Launch
5. Try to rotate the parts, the UI (both IR panel and the right click context menu on the parts) indicate that they are rotating but visually nothing is happening
### Log

In KSP.log, there is the following error message

```
[LOG 20:59:27.002] Vessel assembly complete!
[LOG 20:59:27.002] all systems started
[LOG 20:59:27.030] IR: Loading sounds : MagicSmokeIndustries/Sounds/infernalRoboticMotor
[LOG 20:59:27.030] IR: Sound successfully loaded.
[ERR 20:59:27.031] Cannot find module 'ModuleIRServo' (291586916)

[EXC 20:59:27.032] NullReferenceException: Object reference not set to an instance of an object
    InfernalRobotics.Module.ModuleIRServo.<IsSymmMaster>b__131_0 (.Part cp)
    System.Linq.Enumerable.All[Part] (IEnumerable`1 source, System.Func`2 predicate)
    InfernalRobotics.Module.ModuleIRServo.IsSymmMaster ()
    InfernalRobotics.Module.ModuleIRServo.AttachToParent ()
    InfernalRobotics.Module.ModuleIRServo.BuildAttachments ()
    InfernalRobotics.Module.ModuleIRServo.OnStart (StartState state)
    Part.ModulesOnStart ()
    Part+<Start>c__Iterator25.MoveNext ()
[LOG 20:59:27.035] IR: Loading sounds : MagicSmokeIndustries/Sounds/infernalRoboticMotor
[LOG 20:59:27.035] IR: Sound successfully loaded.
[ERR 20:59:27.036] Cannot find module 'ModuleIRServo' (291586916)

[EXC 20:59:27.037] NullReferenceException: Object reference not set to an instance of an object
    InfernalRobotics.Module.ModuleIRServo.<IsSymmMaster>b__131_0 (.Part cp)
    System.Linq.Enumerable.All[Part] (IEnumerable`1 source, System.Func`2 predicate)
    InfernalRobotics.Module.ModuleIRServo.IsSymmMaster ()
    InfernalRobotics.Module.ModuleIRServo.AttachToParent ()
    InfernalRobotics.Module.ModuleIRServo.BuildAttachments ()
    InfernalRobotics.Module.ModuleIRServo.OnStart (StartState state)
    Part.ModulesOnStart ()
    Part+<Start>c__Iterator25.MoveNext ()
```

**Comment by ZiwKerman** [comment]
Thank you. It is known, but you are actually the first one to report it properly with relevant logs, so kudos to you, this will save me a lot of time needed to fix it. I'll be fixed in the next update coming next week (Wed or Thu)

**Comment by davv1** [comment]
I will add something from my experience: In VAB/SPH they are rotating/moving. Only in Sim they aren't.

**Comment by ZiwKerman** [comment]
Would be fixed once #95 is released later today.
